recipes/simage/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "simage" %}
+{% set version = "1.7.0" %}
+{% set sha256 = "4b57ed3c4997fbcc8b13823a60c1115cd97f701ada8afd44241f8512a5a9adc0" %}
+{% set download_url = "https://github.com/FreeCAD/pivy/releases/download/0.6.3/simage.tar.gz" %}
```

Comment:
Please use the official release for simage

Reply:
there is no official release for the cmake-branch of simage.

Reply:
Can you use `https://bitbucket.org/Coin3D/simage/get/cf953eacd849.tar.gz`? It's the latest master. Also you can do it like
`{% set commit = "cf953eacd849" %}` and make the url `https://bitbucket.org/Coin3D/simage/get/{{ commit }}.tar.gz` below